assets/panel/debugger.properties

```diff
@@ -494,13 +494,30 @@ editor.editBreakpoint=Edit breakpoint
 
 # LOCALIZATION NOTE (editor.addConditionalBreakpoint): Editor gutter context
 # menu item for adding a breakpoint condition on a line.
-editor.addConditionalBreakpoint=Add conditional breakpoint
+editor.addConditionalBreakpoint=Add condition
 editor.addConditionalBreakpoint.accesskey=c
 
+# LOCALIZATION NOTE (editor.editBreakpoint): Editor gutter context menu item
+# for setting a breakpoint condition on a line.
+editor.editConditionalBreakpoint=Edit condition
+
+# LOCALIZATION NOTE (editor.addLogPoint): Editor gutter context
+# menu item for adding a log point on a line.
+editor.addLogPoint=Add log
+editor.addLogPoint.accesskey=l
+
+# LOCALIZATION NOTE (editor.editLogpoint): Editor gutter context menu item
+# for setting a log point on a line.
```

Comment:
This would benefit from a comment explaining that "Edit" is a verb, and this doesn't mean "Log of edits".